srcpkgs/v2ray-domain-list-community/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'v2ray-domain-list-community'
+pkgname=v2ray-domain-list-community
+version=20220809072217
+revision=1
+wrksrc="domain-list-community-${version}"
+build_style=go
+go_import_path="github.com/v2fly/domain-list-community"
+short_desc="Community managed domain list for Project V"
+maintainer="ketlrznt <tansuanyinliao8888@gmail.com>"
+license="MIT"
+homepage="https://github.com/v2fly/domain-list-community"
+distfiles="https://github.com/v2fly/domain-list-community/archive/refs/tags/${version}.tar.gz"
+checksum=3583a9dd91ee9d2f514e5ee49360c579ae52d2a3b1f4693ebd6bc0e819cabfe0
+
+post_install() {
+	local binary_file_path=$(find ${GOPATH}/bin/ -name domain-list-community)
+	$binary_file_path --datapath=${wrksrc}/data --outputname geosite.dat
```

Comment:
I think this line leads to build failures in CI. xbps-src builds binary file successfully, but what I want is the `dat` file generates by go build binary output.

Reply:
A method to solve this is fetching `dat` file released by upstream directly,  But in this way the build process doesn't make sense.

Reply:
1. I think what's happening is you're trying to run a binary built for not-x86_64 on x86_64, which will only work in a few cases.
2. yes, I think fetching the released dat file is better, and can probably be simply added as a distfile for v2ray (but I'm not sure, I don't know anything about this software)

Reply:
> 1. I think what's happening is you're trying to run a binary built for not-x86_64 on x86_64, which will only work in a few cases.
> 
>     2. yes, I think fetching the released dat file is better, and can probably be simply added as a distfile for v2ray (but I'm not sure, I don't know anything about this software)

Done. Thank you for your advice.

For list point 2, `v2ray-domain-list-community` gets updates very rapidly, as upstream releases new dat file every week. It's better to make a separated package for it, or v2ray may gets revision very often, which doesn't mean too much.

Reply:
does the list need to be updated every week? either way you're going to be doing updates very often if so

Reply:
> does the list need to be updated every week? either way you're going to be doing updates very often if so


I'd like to take this responsibility. But frankly speaking, a somewhat outdated list can also provides functionalities. I will update this template as soon as possible after new upstream release.

Reply:
> I'd like to take this responsibility

do remember that someone from the maintainer team would have to review and merge the request each week

Reply:
> > I'd like to take this responsibility
> 
> do remember that someone from the maintainer team would have to review and merge the request each week
I will. Another way is to make the update cycle parallel to v2ray (if possible), I will do some works to test this file and make everything works fine. May not disturb maintainers very often then.